x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -60,12 +62,13 @@ export function AlertsPage({ routeParams }: AlertsPageProps) {
   // observability. For now link to the settings page.
   const manageRulesHref = prepend('/app/management/insightsAndAlerting/triggersActions/alerts');
 
-  const { data: dynamicIndexPatternResp } = useFetcher(({ signal }) => {
+  const { data: indexNames = NO_INDEX_NAMES } = useFetcher(({ signal }) => {
     return callObservabilityApi({
       signal,
       endpoint: 'GET /api/observability/rules/alerts/dynamic_index_pattern',
       params: {
         query: {
+          namespace: 'default',
```

Comment:
@weltenwort Is there a way to get the space id from the UI? This may not always be `default`, no?

We were pulling it [off of the rulesClient](https://github.com/elastic/kibana/pull/109346/commits/3b18c32f74960c0299c3ca202d72c47a4ab4683e#diff-36a9bea00d4b12e6c2de7e0a561c18161cfbe44f6cdb2c684b23d62d1f74ccc0L24) in the `rules/alerts/dynamic_index_pattern` route so we could ensure we acquired the correct space id on the request.. If observability doesn't plan on segmenting by space id I could understand why this would be hard coded to `default` then. Just looking for clarification here..

Reply:
This is intentionally fixed as `default`, because in observability we're not using the space id in the namespace. The filtering by space happens as part of the RBAC filter if I saw that correctly.